src/molecules/chat-ui/index.tsx

```diff
@@ -1,97 +1,97 @@
-import Chat from '@samagra-x/chatui';
-import '@samagra-x/chatui/dist/index.css';
+import Chat from '@samagra-x/chatui'
+import '@samagra-x/chatui/dist/index.css'
 import React, {
   ReactElement,
   useCallback,
   useEffect,
   useMemo,
   useState,
-} from 'react';
+} from 'react'
 import styles from './index.module.css'
-import { getMsgType } from './utils/getMsgType';
-import MessageItem from '../message-item';
-import toast from 'react-hot-toast';
-import { recordUserLocation } from './utils/location';
-import chatHistory from './chatHistory.json';
-import config from './config.json';
-import ShareButtons from '../share-buttons';
+import { getMsgType } from './utils/getMsgType'
+import MessageItem from '../message-item'
+import toast from 'react-hot-toast'
+import { recordUserLocation } from './utils/location'
+import chatHistory from './chatHistory.json'
+import { component, theme } from './../../../app.config.json'
```

Comment:
create a hook similar to whitelabel bot to get this

Reply:
done